src/util.py

```diff
@@ -134,6 +134,7 @@ class Logger:
     dbg_model: bool = False
     dbg_coverage: bool = False
     dbg_generator: bool = False
+    dbg_traces_no_ctr: bool = False
```

Comment:
Probably having only "positive" flags would make things a bit clearer

Reply:
good point. on it

Reply:
fixed